app/forms/submissions/missing_data_validator.rb

```diff
@@ -2,16 +2,30 @@
 
 module Submissions
   class MissingDataValidator < BaseValidator
+    class_attribute :extra_validators, instance_writer: false, default: {}
+
+    class << self
+      def missing_data_validator(name, options)
+        extra_validators[name] = options
+      end
+    end
+
+    missing_data_validator :trainee_start_date, form: "TraineeStartDateForm", if: :course_already_started?
+
     def missing_fields
       forms.map(&:missing_fields).flatten.uniq
     end
 
-    def forms
-      validator_keys.map { |key| validator(key) }
+    def course_already_started?
+      !trainee.course_start_date&.future?
```

Comment:
```suggestion
!trainee.starts_course_in_the_future?
```

https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/models/trainee.rb#L321